### PR TITLE
Add "Mobile Developer Morning Fun" post.

### PR DESCRIPTION
**Pull request by arturdryomov**
@artem-zinnatullin, @dmitry-novikov, PTAL.

You can press the View button to show the GitHub-rendered Markdown.

### CODE REVIEW

**Review by dmitry-novikov — APPROVED** [review]

File: content/posts/2018-06-16-morning-fun.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+# Going Mobile
+
+The thing is, mobile developers already live in the _You Build It — You Run It_ reality.
+Mostly because such teams are usually small or medium-sized.
+There is no infrastructure needed to be run since
+Google and Apple do that themselves. Plus, it is rare to see a dedicated support team
+responding to users. As a result, the development-running loop is pretty tight.
+Developers often oversee the development process itself, deployment and running.
+
+At the same time, [we can do better](https://www.youtube.com/watch?v=SLILjDx0SO0).
```
Comment (artem-zinnatullin) [inline content/posts/2018-06-16-morning-fun.md:83]: expected a rickroll

File: content/posts/2018-06-16-morning-fun.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+# On-call
+
+> :book: Reading material on the topic:
+> [_Testing in Production_](https://medium.com/@copyconstruct/testing-in-production-the-safe-way-18ca102d0ef1) and
+> [_On-call Doesn’t Have to Suck_](https://medium.com/@copyconstruct/on-call-b0bd8c5ea4e0).
+
+Internally we call it _Daily Duty_ because...
+
+* On-call actions are performed in a 24-hour window.
```
Comment (artem-zinnatullin) [inline content/posts/2018-06-16-morning-fun.md:93]: So, this is where people like me would be heated up :)

You can clarify that in your case it doesn't mean that you need to react at night or on weekends.

**Review by artem-zinnatullin — APPROVED** [review]
painful topic for me heh

File: content/posts/2018-06-16-morning-fun.md (hunk 2 above)
Comment (arturdryomov) [reply content/posts/2018-06-16-morning-fun.md:93]: Mentioned it, thanks!

**Pull request merged**